bundle/ContentForms/FieldType/Mapper/BinaryFileFormMapper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\ContentForms\FieldType\Mapper;
+
+use Ibexa\ContentForms\FieldType\DataTransformer\BinaryFileValueTransformer;
+use Ibexa\ContentForms\Form\Type\FieldType\BinaryFileFieldType;
+use Ibexa\Contracts\ContentForms\Data\Content\FieldData;
+use Ibexa\Contracts\ContentForms\FieldType\FieldValueFormMapperInterface;
+use Ibexa\Contracts\Core\Repository\FieldTypeService;
+use Ibexa\Core\FieldType\BinaryFile\Value;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * Overridden to properly handle null value for information collection.
+ */
+class BinaryFileFormMapper implements FieldValueFormMapperInterface
+{
+    private FieldTypeService $fieldTypeService;
+
+    public function __construct(FieldTypeService $fieldTypeService)
+    {
+        $this->fieldTypeService = $fieldTypeService;
+    }
+
+    /**
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
```

Comment:
I always wondered why you put `@throws` tags for methods that do not directly throw? In other words, I believe `@throws` tags should be used only for directly thrown exceptions.

What if calling code changes the exception or adds another one? We certainly do not expect to update it here every time this happens?

Reply:
In to know what is handled and not handled. It's a habit I guess, but yes, it can be irritating.
Removed in https://github.com/netgen/site-bundle/pull/34/commits/e5673938f51b1b3071dca7fed25d29144304408c, since this is just a bug override and it's not in the original implementation.